### ISSUE TITLE
Accessibility ready, mobile navigation isnt working

### ISSUE THREAD

**Issue by Joomlaplates** [templaza/astroid-framework#1191]
Hello Sonny, In Europe there is now a law that your website must be Accessibility ready. That means you have to control the website by keyboard and image alt tags must be there and so on.

**Astroid is working so far so good, but only one thing is not working.**
1. I cant navigate to the offcanvas with the TAB key
2. Mobile menu navigation is not working with the TAB keyboard

Maybe you can take a look at these points. But no rush!
Thanks in advance, Joomlaplates

**Comment by sonvnn** [comment]
Thank you! I will research about it.

**Comment by Vanbrugg** [comment]
It would be helpful, if we could configure things like "tabindex", "data-landmark-index" for all sections/areas in the layout manager.
All elements should also have options for "tabindex". Navigation elements should contain own options for aria attributes (for individual configuration. For links the following attributes missing: "title", "aria-label".

A little suggestion for you to test: You could use the browser add-on "WAVE" (https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) and check errors and other mentions.

**Comment by Joomlaplates** [comment]
Btw I cant reach the Offcanvas with the TAB key.
Just additionally.
